docker/install.sh

```diff
@@ -73,19 +63,37 @@ esac
 shift
 done
 
-# Verify that the driver is provided
+# Set the defaults
 if [[ "$DOCKER_DRIVER" == "" ]]; then
-    echo "Argument -d | --driver is required. Please see options from docker-machine:"
-    echo ""
-    echo $(docker-machine create --help | grep "\-\-driver")
-    exit 1
+    DOCKER_DRIVER=${DEFAULT_DOCKER_DRIVER}
```

Comment:
HINT: on bash 4 there's a format to set the default on a variable and print a message, but this is fine too.